services/supabase/.gitignore

```diff
@@ -3,3 +3,4 @@ volumes/db/init/data.sql
 volumes/storage
 app
 test.http
+services/supabase/.env
```

Comment:
```suggestion

```

If I'm reading this right, this line would ignore `services/supabase/services/supabase/.env` which does not exist. Shouldn't it already be ignored by the gitignore in the repo root?

Reply:
Oh right that was from when I was trying to gitignore before I knew that files already on the commit history would still get committed. Should be good to remove that now.